daemon/cmd/kube_proxy_replacement.go

```diff
@@ -62,6 +62,14 @@ func initKubeProxyReplacementOptions() error {
 		return nil
 	}
 
+	if option.Config.KubeProxyReplacement == option.KubeProxyReplacementPartial {
```

Comment:
can this be merged into 1 if statement? instead of a nested if?

Reply:
It can, but IMO this reads better in the context, which is formed with similar if statements.